sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/tests/Microsoft.OpenTelemetry.Exporter.AzureMonitor.Integration.Tests/TelemetryItemTests.cs

```diff
@@ -63,6 +66,28 @@ public void VerifyActivity(ActivityKind activityKind)
                 });
         }
 
+        [Theory]
+        [InlineData(LogLevel.Information, "Information")]
+        [InlineData(LogLevel.Warning, "Warning")]
+        [InlineData(LogLevel.Error, "Error")]
+        [InlineData(LogLevel.Critical, "Critical")]
+        [InlineData(LogLevel.Debug, "Verbose")]
+        [InlineData(LogLevel.Trace, "Verbose")]
+        public void VerifyILogger(LogLevel logLevel, string expectedSeverityLevel)
+        {
+            var message = "Hello World!";
+
+            var telemetryItem = this.RunLoggerTest(x => x.Log(logLevel: logLevel, message: message));
```

Comment:
perhaps add a test where an Activity is present, so you can validate that TraceId,SpanId is correctly populated.

Reply:
Let me address this in a second PR.